<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-O6A5C2O" failures="8" tests="110" name="com.tetherfi.test.chat.ChatTemplatesTest" time="4559.952" errors="5" timestamp="2020-02-28T13:57:19 IST" skipped="0">
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="ChatTemplatePage" time="33.850"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentsDropdownForAllTheColumns" time="35.381"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentsColumnsHeaderEnable" time="37.093"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentsColumnsHeaderDisable" time="38.971"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsDropdownForAllTheColumns" time="41.682"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsColumnsHeaderEnable" time="40.181"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsColumnsHeaderDisable" time="40.630"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyMainsDropdownForAllTheColumns" time="45.934"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyMainsColumnsHeaderEnable" time="39.742"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyMainsColumnsHeaderDisable" time="43.125"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddNewDepartmentRecord" time="36.880"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddDuplicateDepartmentRecord" time="36.063"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForCreateDepartment" time="55.950"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddEmptyDepartmentRecord" time="33.482"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutDepartment" time="37.326"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutEnabled" time="35.336"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutChannel" time="34.327"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtAddDepartmentRecord" time="52.994"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddNewGroupsRecord" time="121.842">
    <error message="Expected condition failed: waiting for text (&#039;Record Created Successfully&#039;) to be present in element Proxy element for: DefaultElementLocator &#039;By.cssSelector: .toast-message&#039; (tried for 20 second(s) with 500 milliseconds interval)" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for text ('Record Created Successfully') to be present in element Proxy element for: DefaultElementLocator 'By.cssSelector: .toast-message' (tried for 20 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at com.tetherfi.pages.BasePage.waitUntilTextToBePresentInWebElement(BasePage.java:146)
at com.tetherfi.pages.ChatTemplatesPage.verifyNewRecordCreated(ChatTemplatesPage.java:479)
at com.tetherfi.test.chat.ChatTemplatesTest.AddNewGroupsRecord(ChatTemplatesTest.java:269)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":".toast-message"}
  (Session info: chrome=78.0.3904.70)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-O6A5C2O', ip: '10.0.75.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '9.0.4'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 78.0.3904.70, chrome: {chromedriverVersion: 78.0.3904.105 (60e2d8774a81..., userDataDir: C:\Users\admin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:59260}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: d51e0982402a2dc00b875c2ffa6f71c6
*** Element info: {Using=css selector, value=.toast-message}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByCssSelector(RemoteWebDriver.java:420)
at org.openqa.selenium.By$ByCssSelector.findElement(By.java:431)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy14.getText(Unknown Source)
at org.openqa.selenium.support.ui.ExpectedConditions$12.apply(ExpectedConditions.java:353)
at org.openqa.selenium.support.ui.ExpectedConditions$12.apply(ExpectedConditions.java:349)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
... 31 more
]]>
    </error>
  </testcase> <!-- AddNewGroupsRecord -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddDuplicateGroupsRecord" time="44.097"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForCreateGroup" time="54.350"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddEmptyGroupRecord" time="41.187"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutDepartmentName" time="35.166"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutName" time="38.028"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutGroupEnabled" time="37.770"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtAddGroupsRecord" time="53.612"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddNewChatTemplatesRecord" time="118.476">
    <error message="Expected condition failed: waiting for text (&#039;Record Created Successfully&#039;) to be present in element Proxy element for: DefaultElementLocator &#039;By.cssSelector: .toast-message&#039; (tried for 20 second(s) with 500 milliseconds interval)" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for text ('Record Created Successfully') to be present in element Proxy element for: DefaultElementLocator 'By.cssSelector: .toast-message' (tried for 20 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at com.tetherfi.pages.BasePage.waitUntilTextToBePresentInWebElement(BasePage.java:146)
at com.tetherfi.pages.ChatTemplatesPage.verifyNewRecordCreated(ChatTemplatesPage.java:479)
at com.tetherfi.test.chat.ChatTemplatesTest.AddNewChatTemplatesRecord(ChatTemplatesTest.java:357)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":".toast-message"}
  (Session info: chrome=78.0.3904.70)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-O6A5C2O', ip: '10.0.75.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '9.0.4'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 78.0.3904.70, chrome: {chromedriverVersion: 78.0.3904.105 (60e2d8774a81..., userDataDir: C:\Users\admin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:59527}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: a9aa7f22d660d091494250d1a8bd7d67
*** Element info: {Using=css selector, value=.toast-message}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByCssSelector(RemoteWebDriver.java:420)
at org.openqa.selenium.By$ByCssSelector.findElement(By.java:431)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy14.getText(Unknown Source)
at org.openqa.selenium.support.ui.ExpectedConditions$12.apply(ExpectedConditions.java:353)
at org.openqa.selenium.support.ui.ExpectedConditions$12.apply(ExpectedConditions.java:349)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
... 31 more
]]>
    </error>
  </testcase> <!-- AddNewChatTemplatesRecord -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddDuplicateChatTemplatesRecord" time="41.766"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForCreate" time="53.715"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddEmptyChatTemplateRecord" time="38.928"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutChatDepartmentName" time="38.380"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutGroupsName" time="42.520"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutChatName" time="41.052"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutText" time="43.746"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutStartTime" time="43.224"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutEndTime" time="42.922"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="AddRecordWithoutChatEnabled" time="44.029"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtAddChatTemplateRecord" time="55.832"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DepartmentExportToExcel" time="30.771"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DepartmentExportToExcelData" time="41.137"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DepatmentSortingByAscending" time="38.518"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DepartmentSortingByDescending" time="38.531"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="GroupsExportToExcel" time="46.514"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="GroupsExportToExcelData" time="27.566">
    <error message="1" type="java.lang.ArrayIndexOutOfBoundsException">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1
at com.tetherfi.pages.ChatTemplatesPage.getGroupsData(ChatTemplatesPage.java:1064)
at com.tetherfi.pages.ChatTemplatesPage.verifyexportToExcelGroupsSheet(ChatTemplatesPage.java:1113)
at com.tetherfi.test.chat.ChatTemplatesTest.GroupsExportToExcelData(ChatTemplatesTest.java:525)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- GroupsExportToExcelData -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="GroupsSortingByAscending" time="36.486"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="GroupsSortingByDescending" time="40.525"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="ExportToExcel" time="34.936"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="ExportToExcelData" time="32.442"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="SortingByAscending" time="35.856"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="SortingByDescending" time="31.890"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentArrowMoveForPreviousAndNextPage" time="26.403"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentArrowMoveForFirstAndLastPage" time="27.362"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentTotalNumberOfItemsPerPageDetails" time="36.816">
    <failure message="item per page assertion failed did not expect to find [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: item per page assertion failed did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyDepartmentTotalNumberOfItemsPerPageDetails(ChatTemplatesTest.java:600)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyDepartmentTotalNumberOfItemsPerPageDetails -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentNumberOfItemsPerPageSelection" time="31.980">
    <failure message="item per page assertion failed did not expect to find [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: item per page assertion failed did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyDepartmentNumberOfItemsPerPageSelection(ChatTemplatesTest.java:607)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyDepartmentNumberOfItemsPerPageSelection -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsArrowMoveForPreviousAndNextPage" time="32.080"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsArrowMoveForFirstAndLastPage" time="31.861"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsTotalNumberOfItemsPerPageDetails" time="32.052">
    <failure message="item per page assertion failed did not expect to find [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: item per page assertion failed did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyGroupsTotalNumberOfItemsPerPageDetails(ChatTemplatesTest.java:628)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyGroupsTotalNumberOfItemsPerPageDetails -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsNumberOfItemsPerPageSelection" time="33.282">
    <failure message="item per page assertion failed did not expect to find [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: item per page assertion failed did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyGroupsNumberOfItemsPerPageSelection(ChatTemplatesTest.java:635)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyGroupsNumberOfItemsPerPageSelection -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyArrowMoveForPreviousAndNextPage" time="31.624"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyArrowMoveForFirstAndLastPage" time="30.317"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyTotalNumberOfItemsPerPageDetails" time="33.278"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyNumberOfItemsPerPageSelection" time="29.350">
    <failure message="item per page assertion failed did not expect to find [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: item per page assertion failed did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyNumberOfItemsPerPageSelection(ChatTemplatesTest.java:659)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyNumberOfItemsPerPageSelection -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentDragAndDrop" time="34.874"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDepartmentDragAndDropofAlreadyGroupedHeader" time="35.833"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsDragAndDrop" time="31.645"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyGroupsDragAndDropofAlreadyGroupedHeader" time="34.555"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDragAndDrop" time="34.112"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDragAndDropofAlreadyGroupedHeader" time="34.831"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="EditChatTemplatesRecord" time="41.310"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForUpdate" time="55.514"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="EditWithoutModifyReasonRecord" time="41.595"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtEditRecord" time="60.110"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="EditGroupRecord" time="38.955"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForGroupUpdate" time="55.430"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="EditWithoutModifyReasonGroupRecord" time="36.834"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtEditGroupRecord" time="61.218"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyUpdateGroupRecordInMain" time="60.107"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="EditDepartmentRecord" time="43.499"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForDepartmentUpdate" time="54.744"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="EditWithoutModifyReasonDepartmentRecord" time="36.497"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtEditDepartmentRecord" time="57.227"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyUpdateDepartmentRecordInGroup" time="32.904"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyUpdateDepartmentRecordInMain" time="31.814"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="searchDepartmentPage" time="30.575"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="searchDepartmentwithoutSearchTextbox" time="36.725"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="searchGroupsPage" time="31.491"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="searchGroupwithoutSearchTextbox" time="30.270"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="searchPage" time="37.397"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="searchwithoutSearchTextbox" time="33.568"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DeleteWithoutDeleteReasonRecord" time="32.477"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtDeleteChatTemplateRecord" time="59.523"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DeleteChatTemplatesRecord" time="48.018"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForDelete" time="53.373">
    <failure message="did not expect to find [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyAuditTrialReportForDelete(ChatTemplatesTest.java:1004)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyAuditTrialReportForDelete -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DeleteGroupWithoutDeleteReasonRecord" time="34.172"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtDeleteGroupRecord" time="59.455"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DeleteGroupRecord" time="43.437"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForGroupDelete" time="55.928">
    <failure message="did not expect to find [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyAuditTrialReportForGroupDelete(ChatTemplatesTest.java:1052)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyAuditTrialReportForGroupDelete -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDeleteGroupRecordInMain" time="38.363">
    <error message="element click intercepted: Element &lt;span title=&quot;&quot; class=&quot;k-widget k-dropdown&quot; unselectable=&quot;on&quot; role=&quot;listbox&quot; aria-haspopup=&quot;true&quot; aria-expanded=&quot;false&quot; aria-owns=&quot;GroupName_listbox&quot; aria-live=&quot;polite&quot; aria-disabled=&quot;true&quot; style=&quot;width: 100%;&quot; aria-activedescendant=&quot;bac2dfb8-d924-41e2-a443-e4a7c0b9f79d&quot;&gt;...&lt;/span&gt; is not clickable at point (983, 180). Other element would receive the click: &lt;li tabindex=&quot;-1&quot; role=&quot;option&quot; unselectable=&quot;on&quot; class=&quot;k-item&quot; aria-selected=&quot;false&quot; data-offset-index=&quot;0&quot;&gt;...&lt;/li&gt;
  (Session info: chrome=78.0.3904.70)
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;DESKTOP-O6A5C2O&#039;, ip: &#039;10.0.75.1&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;9.0.4&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 78.0.3904.70, chrome: {chromedriverVersion: 78.0.3904.105 (60e2d8774a81..., userDataDir: C:\Users\admin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:61714}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 7d912dfefeeae6544374a7bb3e573bd3" type="org.openqa.selenium.ElementClickInterceptedException">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <span title="" class="k-widget k-dropdown" unselectable="on" role="listbox" aria-haspopup="true" aria-expanded="false" aria-owns="GroupName_listbox" aria-live="polite" aria-disabled="true" style="width: 100%;" aria-activedescendant="bac2dfb8-d924-41e2-a443-e4a7c0b9f79d">...</span> is not clickable at point (983, 180). Other element would receive the click: <li tabindex="-1" role="option" unselectable="on" class="k-item" aria-selected="false" data-offset-index="0">...</li>
  (Session info: chrome=78.0.3904.70)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-O6A5C2O', ip: '10.0.75.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '9.0.4'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 78.0.3904.70, chrome: {chromedriverVersion: 78.0.3904.105 (60e2d8774a81..., userDataDir: C:\Users\admin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:61714}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 7d912dfefeeae6544374a7bb3e573bd3
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at com.sun.proxy.$Proxy14.click(Unknown Source)
at com.tetherfi.pages.BasePage.selectWebElement(BasePage.java:202)
at com.tetherfi.pages.ChatTemplatesPage.VerifyGroupDropdown(ChatTemplatesPage.java:1405)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyDeleteGroupRecordInMain(ChatTemplatesTest.java:1062)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- VerifyDeleteGroupRecordInMain -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DeleteDepartmentWithoutDeleteReasonRecord" time="36.291"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyCancelBtnAtDeleteDepartmentRecord" time="60.408"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DeleteDepartmentRecord" time="40.180"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyAuditTrialReportForDepartmentDelete" time="53.491">
    <failure message="did not expect to find [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.tetherfi.test.chat.ChatTemplatesTest.VerifyAuditTrialReportForDepartmentDelete(ChatTemplatesTest.java:1109)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyAuditTrialReportForDepartmentDelete -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDeleteDepartmentRecordInGroup" time="34.814"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="VerifyDeleteDepartmentRecordInMain" time="32.700"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="SearchDepartmentClearSearch" time="32.373"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="SearchGroupClearSearch" time="38.689"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="SearchClearSearch" time="36.490"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="DepartmentDatabase" time="29.874"/>
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="GroupDatabase" time="32.052">
    <error message="1" type="java.lang.ArrayIndexOutOfBoundsException">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1
at com.tetherfi.pages.ChatTemplatesPage.getGrouptable(ChatTemplatesPage.java:1636)
at com.tetherfi.pages.ChatTemplatesPage.verifyGroupDatabase(ChatTemplatesPage.java:1628)
at com.tetherfi.test.chat.ChatTemplatesTest.GroupDatabase(ChatTemplatesTest.java:1188)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- GroupDatabase -->
  <testcase classname="com.tetherfi.test.chat.ChatTemplatesTest" name="Database" time="33.044"/>
</testsuite> <!-- com.tetherfi.test.chat.ChatTemplatesTest -->
